### PR TITLE
Module 2 revisions

### PR DESCRIPTION
**Pull request by joelostblom**

### CODE REVIEW

File: chapters/en/module2.md [hunk 1]
```diff
@@ -187,7 +187,7 @@ What about when we
 Before making any plots, what is an important first step in visualizing data?
 
 <choice id="1" >
-<opt text="Looking at the data." correct="true">
+<opt text="Look at the data, think about why you are making the visualization, and draw it out with pen and paper." correct="true">
```
Comment (joelostblom) [inline chapters/en/module2.md:190]: These points are first brought up in a slide in module 1 and the reiterated in modeule 2

File: chapters/en/slides/module2/module2_04.Rmd [hunk 2]
```diff
@@ -111,14 +64,13 @@ with data we already know.
...
+## Read in and glimpse at the data with pandas
 
 ```{python}
 import pandas as pd
 
-
-url = 'https://raw.githubusercontent.com/UofTCoders/workshops-dc-py/master/data/processed/world-data-gapminder.csv'
-gm = pd.read_csv(url, parse_dates=['year'])
+gm = pd.read_csv('data/world-data-gapminder.csv', parse_dates=['year'])
+# TODO remove when we change to HTML or rewrite text as for why top ten
```
Comment (joelostblom) [inline chapters/en/slides/module2/module2_04.Rmd:73]: Note this

File: chapters/en/slides/module2/module2_13.Rmd [hunk 3]
```diff
@@ -56,33 +49,129 @@ both within a bar and between bars.
...
+
+## Creating a separate subplot for each regions facilitate comparisons between them
+
+```{python}
+(alt.Chart(gm2018).mark_bar().encode(
+    x=alt.X('life_expectancy', bin=alt.Bin(maxbins=30)),
+    y='count()',
+    color='region')
+ .properties(width=200, height=150)
+ .facet('region'))#.configure_view(stroke=None)
```
Comment (joelostblom) [inline chapters/en/slides/module2/module2_13.Rmd]: This one errors when converting to md

image

File: chapters/en/slides/module2/module2_04.Rmd (hunk 2 above)
Comment (hfboyce) [reply chapters/en/slides/module2/module2_04.Rmd:73]: Wait Should this not be addressed now?

Comment (joelostblom) [reply chapters/en/slides/module2/module2_04.Rmd:73]: Will address in follow up PR

**Pull request merged**